content/nginxaas-azure/getting-started/create-deployment/deploy-azure-portal.md

```diff
@@ -46,17 +48,44 @@ You can start the NGINXaaS deployment process by visiting the [Create NGINXaaS](
+   | Inbound port rules | Select `None` to disallow inbound access on any port, or choose to allow traffic from one of these common http(s) ports. <br><br> **Note:** This option is only available when specifying a new virtual network as part of the create workflow. If you select an existing virtual network which is associated with a subnet and Network Security Group (NSG), you will need to edit the Inbound security rules to add access for the specific ports you want to allow (for example, ports 80 and 443).|
+   | Apply default NGINX configuration | Confirm that you want your NGINXaaS deployment to be bootstrapped with a default NGINX configuration and a browsable splash page. |
    {{</bootstrap-table>}}
 
+   #### **Notes on subnets:**
+
+   - The minimum subnet size for NGINXaaS is `/27` and is enough for a single NGINXaaS deployment even at large scales.
+   - You can use an existing subnet; make sure that the subnet is delegated to `NGINX.NGINXPLUS/nginxDeployments` before creating a deployment in it. To delegate a subnet to an Azure service, see [Delegate a subnet to an Azure service](https://learn.microsoft.com/en-us/azure/virtual-network/manage-subnet-delegation?source=recommendations#delegate-a-subnet-to-an-azure-service).
+   - The subnet can contain other resources; ensure the subnet size is adequate to scale the NGINXaaS deployment.
+   - You can deploy more than one NGINXaas in the same subnet, however, every deployment in the subnet will share the address space (range of IP addresses that resources can use within the VNet), so ensure the subnet is adequately sized to scale the deployments. If you are more than one NGINXaaS deployment or other resources, a larger subnet, for example, a `/24` is recommended.
+   - You can change the virtual network or subnet for an existing NGINXaaS deployment. If the existing NGINXaaS deployment is using a public IP address, you can change the backend virtual network or subnet. Please make sure that the subnet is delegated to `NGINX.NGINXPLUS/nginxDeployments` before creating a deployment in it.
+   - If your DNS nameservers are configured in the same VNet as your deployment, then you can use those DNS nameservers to resolve the hostname of the upstream servers referenced in your NGINX configuration.
+   - Changes to a virtual network's DNS settings will not be applied automatically to your NGINXaaS deployment. To ensure DNS settings are applied, you must add any custom DNS servers to the VNET's DNS settings before creating an NGINXaaS deployment. As a workaround for existing deployments, we recommend using the [`resolver` directive](https://nginx.org/en/docs/http/ngx_http_core_module.html#resolver) to explicitly specify your name server(s) and the [`resolve` parameter](https://nginx.org/en/docs/http/ngx_http_upstream_module.html#resolve) to automatically re-resolve the domain name of the server without restarting NGINX.
+
+      For example,
+
+      ```nginx
```

Comment:
this section is now feeling too bloated. We have only walked the user through the deployment creation workflow and giving them an example configuration here which they won't get to until they actually deploy NGINXaaS. It might confuse someone reading this guide.

Reply:
This content was in the FAQ.

If it is not relevant here please suggest where we can move it.

Reply:
How about we move this into the `get-help` guide. I think that a better place for us to share this information rather than the overview of deployment creation.

Reply:
@puneetsarna can you clarify how much of this section do you want to move to the Get help doc? I imagine it's not just L71-74

Reply:
I have simplified the notes on subnets no unblock this PR. we can revisit them in a separate task